output/summary.go

```diff
@@ -31,22 +36,41 @@ type Summary struct {
 	NumSkipped int
 }
 
+func (s *Summary) Start(config *plan.Config) error {
```

Comment:
nit - suggest passing the config in when creating the reporter struct, instead of passing it in here.